docs/fundamentals/dashboard/standalone-for-python.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Using Aspire Dashboard with Python Applications
+description: How to use the Aspire Dashboard in a Python application.
+ms.date: 08/27/2024
+ms.topic: tutorial
+ms.author: aapowell
+---
+
+# Tutorial: Using the Aspire Dashboard with Python Applications
+
+The [Aspire dashboard](overview.md) provides a great UI for viewing telemetry, and is available as a standalone container image that can be used with any OpenTelemetry-enabled app. In this article, you'll learn how to:
+
+> [!div class="checklist"]
+>
+> - Start the Aspire dashboard in standalone mode.
+> - Use the Aspire dashboard with a Python application.
+
+## Prerequisites
+
+To complete this tutorial, you need the following:
+
+- [Docker](https://docs.docker.com/get-docker/)
```

Comment:
Why not an OCI-compliant container runtime? We usually specify either Docker or Podman.

Reply:
How would `An OCI-compliant container runtime, such as [Docker](...) or [Podman](...)` read?

Reply:
Or even just:

```suggestion
- [Docker](https://docs.docker.com/get-docker/) or [Podman](https://podman.io/).
```